src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--31">
+      <div class="calendar__date-1 calendar_start-day_tue"></div>
+      <div class="calendar__date-2"></div>
```

Comment:
You shouldn't add numbers to each element, use `:nth-child` in styles
```suggestion
      <div class="calendar__date"></div>
```